LayoutTests/imported/w3c/web-platform-tests/trusted-types/eval-csp-tt-default-policy-mutate-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+PASS eval of string where default policy mutates value throws.
+PASS indirect eval of string where default policy mutates value throws.
+FAIL Function constructor with string where default policy mutates value throws. assert_throws_js: function "_ => new Function('return 1+1')" did not throw
```

Comment:
Why is this failing?

Reply:
That isn't addressed by this PR a followup will address that functionality.